GAP_ANALYSIS.md

```
@@ -1,164 +1,223 @@
+Analysis date: 2026-03-29 (updated)
 Protocol reference: Binder1.txt (MS-ASAIRS, MS-ASCAL, MS-ASCMD, MS-ASPROV,
-MS-ASWBXML, MS-OXWSCAL, MS-OXWSCORE, MS-OXWSFOLD, MS-OXWSSYNC, and related specs)
+MS-ASWBXML, MS-ASHTTP, MS-ASDTYPE, MS-OXWSCAL, MS-OXWSCORE, MS-OXWSFOLD,
+MS-OXWSSYNC, MS-OXWSMTGS, MS-OXWSADISC, MS-OXWSCDATA, and related specs)
 
 ---
 
-## CRITICAL — Code Will Not Compile
+## PREVIOUS GAPS (Session 1 — all carried over)
 
 ### GAP-01 `src/timezone.rs` is missing
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)